packages/expo/bundledNativeModules.json

```
@@ -91,6 +91,10 @@
   "expo-video-thumbnails": "~6.2.1",
   "expo-web-browser": "~10.1.0",
   "lottie-react-native": "5.0.1",
+  "react": "17.0.2",
+  "react-dom": "17.0.2",
+  "react-native": "0.68.0",
+  "react-native-web": "0.17.1",
```

What about `~`? Would be great to not spam too much if someone wants to have different patch version that should also work fine.

```suggestion
  "react": "~17.0.2",
  "react-dom": "~17.0.2",
  "react-native": "~0.68.0",
  "react-native-web": "~0.17.1",
```

---

These are based on the existing versions from the templates [here](https://github.com/expo/expo/blob/6cd5872764bc13d57c2ae124d15f7befb6f59091/templates/expo-template-blank/package.json#L16-L19).

---

So perhaps we should update it in both places, especially that we're going to use `react-native@0.68.1` instead of `0.68.0`